### DISCUSSION TITLE
Wordy

### DISCUSSION THREAD

**Discussion by HimoriK** [beejjorgensen/bgc#132] [Ideas]
@beejjorgensen This book is excessively wordy, although that's the teaching style it seems to detract from the impact of the information.

Consider condensing or removing some paragraphs, as the book could be reduced by half. *I had to stop reading it* because of one such paragraph ```You’re passing one argument to the function printf(): a string to be printed when you call it. Oh, yeah—
we’re calling a function! We rock! Wait, wait—don’t get cocky. What’s that crazy \n at the end of the string?
Well, most characters in the string will print out just like they are stored..``` etc.

Unless you recommend only your v2 reference, but your network socket book is closer to what this could be.

**Comment by veganaiZe** [comment]
If you read txt file tutorials on BBSs in the early 90's then you'd totally get it.

_RoCk On!_

**Comment by beejjorgensen** [comment]
I'll consider it, but this is the first time I've gotten this particular feedback, so it might have to wait a bit. :)
